cloudbuild.yaml

```
@@ -1,94 +1,180 @@
+          "gs://$_CDN_BUCKET/v3/kommunicate.app" || true
+        
+        # 4️ HTML entry file (VERY IMPORTANT)
+        gsutil setmeta \
+          -h "Cache-Control:public,max-age=300" \
+          "gs://$_CDN_BUCKET/index.html" || true
+
+      else
+        echo "Skipping CDN cache headers"
+      fi
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (2 > 0)

(empty-lines)